build_badtrack.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+
+import os
+import shutil
+import subprocess
+
+# Get the current working directory
+APP_PATH = os.getcwd()
```

Comment:
`os.getcwd()` changes depends on where you `cd` to. Running `python ./Desktop/badtrack/build_badtrack.py` and `cd Desktop/badtrack;python ./build_badtrack.py` will give you different values.

Best to do:
```python
APP_PATH = os.abspath(os.path.dirname(__file__))
```